data/json/items/gun/40.json

```diff
@@ -207,5 +207,42 @@
       [ "stock", 1 ],
       [ "underbarrel", 1 ]
     ]
+  },
+  {
+    "id": "hi_power_40",
+    "copy-from": "pistol_base",
+    "type": "GUN",
+    "name": "Browning Hi Power .40 S&W",
+    "description": "The Browning Hi Power is a semi automatic handgun. It is one of the most widely used military pistols in history. This can be loaded with .40 S&W.",
+    "weight": 900,
```

Comment:
I hate citing Wikipedia, but weight is listed as 1 kg - I have no idea if that is 'loaded' weight or not - not really an issue, just mentioning it.

Reply:
Wikipedia is really the best place for info on firearms, tbh, even though there should be better places.
From Wikipedia, looking in the actual contents of the article, I presumed 1 kg is the "loaded" weight:

> This last criterion was seen to demand a caliber of 9 mm or larger, a bullet mass of around 8 grams (123.5 grains), and a muzzle velocity of 350 m/s (1148 ft/s). It was to accomplish all of this at a weight not exceeding 1 kg

Reply:
I thought the same thing - but because it wasn't specific - I thought I'd mention it. No harm, no foul.